### PR TITLE
changes the sequence of Provided APIs

### PR DESCRIPTION
**Pull request by umangachapagain**
Brings OCS APIs to the front.

image
Screenshot of OCS 4.3 on OCP 4.4 Console

Signed-off-by: Umanga Chapagain <chapagainumanga@gmail.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -11,6 +11,7 @@ require (
 	github.com/go-logr/logr v0.1.0
 	github.com/go-logr/zapr v0.1.1
 	github.com/go-openapi/spec v0.19.2
+	github.com/go-openapi/validate v0.18.0 // indirect
```
Comment (egafford) [inline go.mod]: The rest of this patch appears to be a simple copy/paste move with no substantive change. Why are we adding a module here?

Comment (umangachapagain) [reply go.mod]: Removed. Was autogenerated for some reason.

Comment (egafford) [reply go.mod]: Thanks! And thanks for the second PR to add the required dep.

**Pull request merged**